src/chrome/create-chrome.ts

```diff
@@ -253,6 +254,8 @@ export const createChromeContext = ({
     enablePackagesDebug: () => warnDuplicatePkg(),
     requestPdf,
     drawerActions,
+    search: searchAPI,
```

Comment:
Something to think about for the future. Do we want to make the search its own thing, as we did with the notifications drawer state.